README.rst

```diff
@@ -96,6 +96,13 @@ You are recommended to update pip before installing aqtinstall.
     Older pip does not handle it expectedly(see #230).
 
 
+It may be difficult to set up some Windows systems with the correct version of Python and all of ``aqt``'s dependencies.
+To get around this problem, ``aqtinstall`` offers ``aqt.exe``, a Windows executable that contains Python and all required dependencies.
+You may access ``aqt.exe`` from the `Releases section`_, under "assets".
```

Comment:
I think it may be better to add a persistent link to a continuous release, like [linuxdeployqt](https://github.com/probonopd/linuxdeployqt/releases/continuous). However, since I'm not sure how to set up continuous builds, I think this will work for now.

Reply:
Hi. I found that [AppImageCrafters/appimage-builder](https://github.com/AppImageCrafters/appimage-builder) uses [`marvinpinto/action-automatic-releases`](https://github.com/marvinpinto/action-automatic-releases) to release continuous builds. Please see the [workflow](https://github.com/AppImageCrafters/appimage-builder/blob/35e3eabf2e9b0b9b0a980591dc4b401c37ec7b28/.github/workflows/continuous-builds.yml#L95-L100) file.

Reply:
I've written a continuous-build workflow recently;
https://github.com/eb4j/furoku-data/blob/main/.github/workflows/build-furoku.yml

It may help

Reply:
Note that the repository of [`johnwbyrd/update-release`](https://github.com/johnwbyrd/update-release) has been archived.